src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/FlagPipelineTab.tsx

```diff
@@ -5,37 +5,84 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import React, { useState } from 'react'
+import React, { useMemo, useState } from 'react'
 import { Container } from '@harness/uicore'
+import { useParams } from 'react-router-dom'
 import { NoData } from '@cf/components/NoData/NoData'
 import imageUrl from '@cf/images/pipeline_flags_empty_state.svg'
 import { useStrings } from 'framework/strings'
+import { useGetFeaturePipeline } from 'services/cf'
+import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
+import { ContainerSpinner } from '@common/components/ContainerSpinner/ContainerSpinner'
 import AvailablePipelinesDrawer from './components/available-pipelines-drawer/AvailablePipelinesDrawer'
-
+import ConfiguredPipelineView from './components/configured-pipeline-view/ConfiguredPipelineView'
 export interface FlagPipelineTabProps {
   flagIdentifier: string
 }
 
 const FlagPipelineTab: React.FC<FlagPipelineTabProps> = ({ flagIdentifier }) => {
   const { getString } = useStrings()
+  const { activeEnvironment: environmentIdentifier } = useActiveEnvironment()
+  const { orgIdentifier, accountId: accountIdentifier, projectIdentifier } = useParams<Record<string, string>>()
+
+  const queryParams = useMemo(
+    () => ({
+      identifier: flagIdentifier,
+      environmentIdentifier,
+      projectIdentifier,
+      accountIdentifier,
+      orgIdentifier
+    }),
+    [accountIdentifier, environmentIdentifier, orgIdentifier, projectIdentifier, flagIdentifier]
+  )
+
+  const {
+    data: featurePipeline,
+    loading,
+    refetch: refetchFeaturePipeline
+  } = useGetFeaturePipeline({
+    identifier: flagIdentifier,
+    queryParams,
+    debounce: 500
+  })
 
   const [isDrawerOpen, setIsDrawerOpen] = useState(false)
+
+  if (loading && !isDrawerOpen)
+    return (
+      <Container height="100%" flex={{ align: 'center-center' }}>
+        <ContainerSpinner />
+      </Container>
+    )
```

Comment:
Defo for multi-line statements make sure to use brackets.
```suggestion
  if (loading && !isDrawerOpen) {
    return (
      <Container height="100%" flex={{ align: 'center-center' }}>
        <ContainerSpinner />
      </Container>
    )
  }
```

Reply:
fixed